lms/static/js/spec/main.js

```diff
@@ -317,6 +318,10 @@
                 exports: 'js/ccx/schedule',
                 deps: ['jquery', 'underscore', 'backbone', 'gettext', 'moment']
             },
+            'js/commerce/views/receipt_view': {
+                exports: 'edx.commerce.ReceiptView',
+                deps: ['jquery', 'underscore', 'underscore.string', 'backbone']
```

Comment:
@ahsan-ul-haq can you please try moving backbone to being after jquery in the array?